README.md

```diff
@@ -1,5 +1,9 @@
 # Dynamic Query Builder Client
 
+[![CircleCI](https://circleci.com/gh/emrekeskinmac/dynamic-query-builder-client/tree/feature%2Funit-tests.svg?style=svg)](https://circleci.com/gh/emrekeskinmac/dynamic-query-builder-client/tree/feature%2Funit-tests)
```

Comment:
I'll provide here oplog account for circle ci as soon as possible

Reply:
We have created circle ci account for oplog. Can you change status badge to the following;
```
[![CircleCI](https://circleci.com/gh/oplog/dynamic-query-builder-client.svg?style=svg)](https://circleci.com/gh/oplog/dynamic-query-builder-client)
```